introduction.md

```diff
@@ -0,0 +1 @@
+! mi name is Stefano eadn i'm trying to do this funking course in the half on the night
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.